static/docs/understanding-dvc/related-technologies.md

```diff
@@ -1,22 +1,24 @@
 # Comparison to Existing Technologies
 
-Due to the the novelty of this approach, DVC can be better understood in
-comparison to existing technologies and ideas.
+Due to the the novelty of our approach, it may be easier to understand DVC in
+comparison to existing technologies and tools.
 
-DVC combines a number of existing technologies and ideas into a single product
-with the goal of bringing the best engineering practices into the data science
-process.
+DVC combines a number of existing ideas into a single product, with the goal of
+bringing best practices from software engineering into the data science field.
 
-1. **Git**. The difference is:
+## Differences with related tools
```

Comment:
it repeats the main title?

Reply:
we can easily do `##` (second level) here instead of numbered list

Reply:
H2 looks too "important" for a series of short sections, it even has a horizontal bar underneath. That's why I wanted to use H3 below (e.g. `### Git`) but I didn't want to skip from H1 to H3 so I came up with `## Differences with related tools` which now that I think about it I agree sounds repetitive. I guess it doesn't matter to skip, I'll just remove it! We're getting rid of this entire section of the docs (hopefully) soon anyway.

Reply:
👍